json-core/src/main/java/io/micronaut/json/codec/JsonCodecMediaTypeCodec.java

```diff
@@ -47,26 +44,27 @@
  * @author svishnyakov
  * @since 1.3.0
  */
-public abstract class JacksonMediaTypeCodec implements MediaTypeCodec {
+public abstract class JsonCodecMediaTypeCodec implements MediaTypeCodec {
```

Comment:
The duplication in the name makes this confusion. Probably should be just called `JsonMediaTypeCodec` or `JsonNodeMediaTypeCodec`